PROCESSORS.md

```diff
@@ -2424,6 +2424,7 @@ In the list below, the names of required properties appear in bold. Any other pr
 | tail-base-directory        |                   |                                                        | Base directory used to look for files to tail. This property is required when using Multiple file mode. Can contain expression language placeholders if Attribute Provider Service is set.<br/>**Supports Expression Language: true**                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | **tail-mode**              | Single file       | Single file<br>Multiple file<br>                       | Specifies the tail file mode. In 'Single file' mode only a single file will be watched. In 'Multiple file' mode a regex may be used. Note that in multiple file mode we will still continue to watch for rollover on the initial set of watched files. The Regex used to locate multiple files will be run during the schedule phrase. Note that if rotated files are matched by the regex, those files will be tailed.                                                                                                                                                                                                                                                                                            |
+| **Batch Size**             | 0                 |                                                        | Maximum number of lines to process in a single trigger. If set to 0 all new lines will be processed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
I think it would make sense to combine this with another property that controls whether a batch is emitted as a single flow file or one flow file per line.

Reply:
If the delimiter already control how the lines are processed, is it okay if we only change the description as @adamdebreceni suggested?

Reply:
I would update the property name as well. This leaves the door open for a future property that controls how many lines are emitted in a single flow file, with a similar same name.

Reply:
After reading the comments I don't think we can really talk about "lines" specifically in this processor, as the flow file contents are fully controlled by the delimiter property. So I'm not sure it would be a good idea to have a separate property that controls lines specifically. The `Batch Size` property name is consistently used in all processors to mark the number of flow files emitted at once by the processor so it should represent the same here.

Reply:
My bad, the Batch Size property indeed means the same in all other processors.